README.md

```diff
@@ -2,9 +2,36 @@
 
 ![Portfolio](images/portfolio.png)
 
+### Description
 
 This is my portfolio website to show employers and hiring managers what am are capable of doing and the projects I have build so far.
 
+
+## Accessibility Report
+<img src="images/accessibility-report.png" height="20%" width="auto">
+
+### This is the accessibility report 
+
+- Page titles - `No issue found.`
+
+- Images text alternatives - `No issues found`
+
+- Headings `No Issues found`git 
+
+- Contrast - `There was an issue with my bottom background , but i have fixed them` [LineFixedIssue]()
+
+- Resize - `There was an issue with page resizing. But it has been fixed`
+
+- Interaction - `There were no labels with the forms. But Labels has been added for screen readers only`
+
+- Moving content - `No issues found`
+
+- Multimedia - `No issues found`
+
+- The basic structure of the page  - `No Issues found`
```

Comment:
Congratulations on your very good work :clap: 

However, this report, as stated in the project requirements, should be in the description of the pull request and not here, in the README file; it is not relevant for a user to read it - users will feel it navigating the app.

Screenshot from project requirements:
![image](https://user-images.githubusercontent.com/83596135/166486188-ee76a598-24e6-49ff-91f8-100895f2d63d.png)


- [ ] Kindly, remove the report from the README but paste it in the description of the pull request.

Reply:
ok
The issues is resolved